string object_name
---
sensor_msgs/PointCloud2 segmented_point_cloud